shared/status.js

```diff
@@ -22,6 +22,14 @@ const jsvuPath = config.path;
 
 const statusFilePath = `${jsvuPath}/status.json`;
 
+const translate = {
+	ch: 'chakra',
+	chakracore: 'chakra',
+	jsshell: 'spidermonkey',
+	d8: 'v8',
+	jsc: 'javascriptcore',
+};
```

Comment:
This should be a Map.

Reply:
I'm afraid I'm not capturing how to best use a Map here, would you mind showing me a quick example? I wonder if you have a suggestion to simplify this and I'm looking forward to learn it.

Reply:
```js
const translate = new Map([
	['ch', 'chakra'],
	['chakracore', 'chakra'],
	['jsshell', 'spidermonkey'],
	['d8', 'v8'],
	['jsc', 'javascriptcore'],
]);
```